gcp_variant_transforms/testing/integration/small_tests/merge_option_copy_filter_to_calls.json

```diff
@@ -0,0 +1,58 @@
+{
+  "test_name": "merge-option-copy-filter-to-calls",
+  "table_name": "merge_option_copy_filter_to_calls",
+  "input_pattern": "gs://gcp-variant-transforms-testfiles/small_tests/merge/*.vcf",
+  "variant_merge_strategy": "MOVE_TO_CALLS",
+  "copy_filter_to_calls": true,
+  "runner": "DataflowRunner",
+  "assertion_configs": [
+    {
+      "query": ["NUM_ROWS_QUERY"],
+      "expected_result": {"num_rows": 4}
+    },
+    {
+      "query": ["SUM_START_QUERY"],
+      "expected_result": {"sum_start": 1283553}
+    },
+    {
+      "query": ["SUM_END_QUERY"],
+      "expected_result": {"sum_end": 1283560}
+    },
+    {
+      "query": [
+        "SELECT COUNT(0) AS num_rows ",
+        "FROM {TABLE_NAME} AS t, t.call as call ",
+        "WHERE start_position = 14369 AND call.name ='NA00001' ",
+        "AND 'q10' IN UNNEST (call.filter)"
+      ],
+      "expected_result": {"num_rows": 1}
```

Comment:
Instead of having 4 queries returning "1" each, what do you think about having a single query and instead of `COUNT`, selects `call.name` with an `ORDER BY` at the end (to make the output unique); for the `expected_results` you then verity the list of call names, i.e., "NA*"?

Reply:
For one thing, I think it is expected to have only one row in the query result. Another concern I have is that if we use `call.name` instead of some queries based on `call.filter`, the test will pass with/without `--copy_filter_to_calls`.